src/integrationTest/kotlin/com/sourcegraph/cody/edit/DocumentCodeTest.kt

```diff
@@ -37,28 +40,45 @@ class DocumentCodeTest : CodyIntegrationTextFixture() {
         selection.startOffset == caret && selection.endOffset == caret)
   }
 
-  fun skip_testGetsWorkingGroupLens() {
+  fun testGetsWorkingGroupLens() {
+    var lenses: LensWidgetGroup? = null
+    var widgets: MutableList<LensWidget>? = null
+    val showWorkingGroupSessionStateListener =
+        object : FixupService.ActiveFixupSessionStateListener {
+          override fun fixupSessionStateChanged(isInProgress: Boolean) {
+            if (isInProgress) {
+              TestCase.assertNull("Value already set", lenses)
+              TestCase.assertNull("Value already set", widgets)
+              lenses = activeSession().lensGroup
+              widgets = lenses?.widgets
+            }
+          }
+        }
+    FixupService.getInstance(project).addListener(showWorkingGroupSessionStateListener)
     runAndWaitForNotifications(DocumentCodeAction.ID, TOPIC_DISPLAY_WORKING_GROUP)
     assertInlayIsShown()
 
     // Lens group should match the expected structure.
-    val lenses = activeSession().lensGroup
     assertNotNull("Lens group should be displayed", lenses)
+    val theWidgets = widgets!!
```

Comment:
Using `the` in variable names seems a bit unusual to me :) But it does not matter too much